packages/react-variant-view/src/component/pathogenicity/ClinVarInterpretation.tsx

```diff
@@ -2,82 +2,88 @@ import * as React from 'react';
 import { observer } from 'mobx-react';
 import featureTableStyle from '../featureTable/FeatureTable.module.scss';
 import { computed, makeObservable } from 'mobx';
-import { ClinVar } from 'genome-nexus-ts-api-client';
-import { Button } from 'react-bootstrap';
+import { Clinvar } from 'genome-nexus-ts-api-client';
 import { DefaultTooltip } from 'cbioportal-frontend-commons';
-import {
-    ClinVarRcvInterpretation,
-    getRcvCountMap,
-    getRcvData,
-} from 'react-mutation-mapper';
 
-export interface IClinVarInterpretationProps {
-    clinVar: ClinVar | undefined;
+export interface IClinvarInterpretationProps {
+    clinvar: Clinvar | undefined;
 }
 
-const ClinVarTooltip = () => {
-    return (
-        <DefaultTooltip
-            placement="top"
-            overlay={<span>ClinVar Interpretation</span>}
-        >
-            <Button bsStyle="link" className="btn-sm p-0">
-                ClinVar Interpretation
-            </Button>
-        </DefaultTooltip>
-    );
-};
-
 @observer
-class ClinVarInterpretation extends React.Component<
-    IClinVarInterpretationProps
+class ClinvarInterpretation extends React.Component<
+    IClinvarInterpretationProps
 > {
-    constructor(props: IClinVarInterpretationProps) {
+    constructor(props: IClinvarInterpretationProps) {
         super(props);
         makeObservable(this);
     }
 
-    @computed get rcvCountMap() {
-        if (this.props.clinVar) {
-            return getRcvCountMap(this.props.clinVar);
-        }
-
-        return undefined;
+    @computed get clinvarTooltip() {
+        const clinvarUrl = this.props.clinvar
+            ? `https://www.ncbi.nlm.nih.gov/clinvar/variation/${this.props.clinvar.clinvarId}/`
+            : `https://www.ncbi.nlm.nih.gov/clinvar/`;
+        return (
+            <DefaultTooltip
+                placement="top"
+                overlay={<span>ClinVar Interpretation</span>}
+            >
+                <a href={clinvarUrl} target="_blank" rel="noopener noreferrer">
+                    ClinVar Interpretation
+                </a>
+            </DefaultTooltip>
+        );
     }
 
-    @computed get rcvDisplayData() {
-        if (this.rcvCountMap) {
-            return getRcvData(this.rcvCountMap);
+    @computed get clinvarData() {
```

Comment:
The name `clinvarData` is a bit confusing because we are returning a `div` here. Let's find another name for this if possible.